linux/Platform.c

```diff
@@ -1104,3 +1105,33 @@ void Platform_done(void) {
    LibSensors_cleanup();
 #endif
 }
+
+
+double Platform_getGpuUsage(Meter* meter) {
+   unsigned int i;
+   uint64_t monotonictimeDelta;
+   const Machine* super = meter->host;
+   const LinuxMachine* this = (const LinuxMachine*) super;
```

Comment:
```suggestion
   const Machine* host = meter->host;
   const LinuxMachine* lhost = (const LinuxMachine*) host;
```

Avoid the names `this` and `super` for non-methds.